linker/linker_test.go

```diff
@@ -35,6 +39,20 @@ import (
 	"github.com/bufbuild/protocompile/reporter"
 )
 
+var (
+	protocPath = "../internal/testdata/protoc/22.0/bin/protoc"
+	nullDevice = "/dev/null"
```

Comment:
Use `os.DevNull` instead. It already handles OS-specific values, like "NUL" for Windows.